lib/models/channel.dart

```diff
@@ -24,25 +24,38 @@ class Channel implements ShareLinks {
   List<String>? allowedRegions;
   List<VideoInList>? latestVideos;
 
-  Channel(this.author, this.authorId, this.authorUrl, this.authorBanners, this.authorThumbnails, this.subCount, this.totalViews, this.joined, this.autoGenerated, this.isFamilyFriendly,
-      this.description, this.allowedRegions, this.latestVideos);
-
-  factory Channel.fromJson(Map<String, dynamic> json) => _$ChannelFromJson(json);
+  Channel(
+      this.author,
+      this.authorId,
+      this.authorUrl,
+      this.authorBanners,
+      this.authorThumbnails,
+      this.subCount,
+      this.totalViews,
+      this.joined,
+      this.autoGenerated,
+      this.isFamilyFriendly,
+      this.description,
+      this.allowedRegions,
+      this.latestVideos);
```

Comment:
Sorry my IDE caught this. Is it okay or should I revert?

Reply:
It's ok. I'll run a project wise format once all the few prs are merged.